application/views/contesting/index.php

```diff
@@ -103,7 +103,8 @@
                             <div class="mb-3 col-md-3">
                                 <label for="callsign"><?php echo lang('gen_hamradio_callsign'); ?></label>
                                 <input type="text" class="form-control form-control-sm" id="callsign" name="callsign" required>
-                                <small id="callsign_info" class="badge text-bg-danger"></small>
+                                <small id="callsign_info" class="badge text-bg-danger"></small><br/>
```

Comment:
just remote the `<br>`. everything else is great :) thx for that one! 

```suggestion
                                <small id="callsign_info" class="badge text-bg-danger"></small>
```

Reply:
rr was able to reproduce that. sorry

Reply:
as chatted, @HB9HIL - if you have an idea how to fix that (and if its important enough), pse open new PR.
I tested with Safari, FF and Chrome (wothout BR) and the result was always bearing direct next to Warning.

As best-effort i'll merge this.